src/AppBundle/Event/EventInvitation.php

```diff
@@ -41,8 +42,69 @@ class EventInvitation
      */
     public $guests = [];
 
+    public static function createFromAdherent(?Adherent $adherent): self
+    {
+        $dto = new self();
+
+        if ($adherent) {
+            $dto->setLastName($adherent->getLastName());
```

Comment:
I don't get why you added getters and setters, I've removed them in https://github.com/EnMarche/en-marche.fr/commit/4b8f8712c66d85c240d86d6c195d1f8038e850dc.